src/toil/utils/toilMain.py

```diff
@@ -29,7 +30,7 @@ def main():
     del sys.argv[1]
     module.main()
 
-def loadModules():
+def loadModules() -> Dict[str, Any]:
```

Comment:
Can you be more specific here?

Reply:
I think these are Python modules in the dict; they don't really have a nice base type, do they?

Reply:
Yeah I don't think there's a nice way to get ahold of the real type, and even if we had it it would tell us nothing useful about the members:

```
[anovak@octagon ~]$ python3 -i
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import collections
>>> type(collections)
<class 'module'>
>>> module
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'module' is not defined
>>> import typing
>>> typing.Module
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'typing' has no attribute 'Module'
>>> typing.module
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'typing' has no attribute 'module'
>>> 
```

Reply:
https://docs.python.org/3/library/types.html#types.ModuleType ??

Reply:
```suggestion
def loadModules() -> Dict[str, types.ModuleType]:
```

and add `import types` at the top of the file

Reply:
When I tried adding in `types.ModuleType` I was running into this:

```
src/toil/utils/toilMain.py:32: error: Module has no attribute "main"
src/toil/utils/toilMain.py:51: error: Item "None" of "Optional[str]" has no attribute "strip"
```

Since ModuleType doesn't have attribute main I'm not sure if we can specify this without producing an error.